ionic2/app/pages/pokedetail/pokedetail.page.ts

```diff
@@ -3,10 +3,10 @@ import { Page, NavController, NavParams } from 'ionic-angular';
 import { NavbarComponent } from '../../components/navbar/navbar.component';
 
 @Page({
-  templateUrl: 'pages/wiki/wiki.page.html',
+  templateUrl: 'pages/pokedetail/pokedetail.page.html',
```

Comment:
To be consistent this file should be named `poke-detail.page.*` and should be placed in the folder `pages/poke-detail`.